src/rendering/src/pages/_app.tsx

```diff
@@ -55,6 +55,24 @@ function App({ Component, pageProps, router }: AppPropsWithLayout): JSX.Element
   });
   // END CUSTOMIZATION
 
+  // DEMO TEAM CUSTOMIZATION - Search SDK integration
+  useEffect(() => {
+    if (isSearchSDKEnabled) {
+      PageController.getContext().setLocaleLanguage('en');
+      PageController.getContext().setLocaleCountry('us');
+      trackEntityPageViewEvent('content', {
```

Comment:
Don't forget to pull latest updates from 'develop' - path.tsx already has tracking.

https://github.com/Sitecore/Sitecore.Demo.XmCloud.PlaySummit/blob/develop/src/rendering/src/pages/%5B%5B...path%5D%5D.tsx